### ISSUE TITLE
The tooltip “Lorem” for “info-circle” button is revealed only by mouse over and is not available to keyboard-only users in Tooltips tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1927]
URL: http://clarity-ks.surge.sh/tooltips

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Tooltips tab.
3. Use TAB key to check the tooltip of “info-circle” button in Tooltips tab.

Actual Results:
The tooltip “Lorem” for “info-circle” button is revealed only by mouse over and is not available to keyboard-only users in Tooltips tab.
For more details, please refer to "Tooltips-NotAvailableOnscreen.png".

Expected Results:
The tooltip “Lorem” for “info-circle” button is revealed only by mouse over and should be available onscreen in Tooltips tab.

Fix Suggestion:
Add equivalent text on screen and the text is consistent with tooltip.
Or the equivalent text can be displayed with TAB key.

Violation of Section 508 Standards: 
1194.21(a) When software is designed to run on a system that has a keyboard, product functions shall be executable from a keyboard where the function itself or the result of performing a function can be discerned textually. 

User Impact: Blind.

Screen shot:
1.	Tooltips-NotAvailableOnscreen.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/7fdcee43-22c9-43ba-b25c-76347a0b4327)

**Comment by mathisscott** [comment]
Verified that ANGULAR tooltips are no longer accessible. See website to test it out.
